src/main/java/net/neoforged/neoforge/common/extensions/IItemExtension.java

```diff
@@ -337,11 +338,23 @@ default ResourceLocation getArmorTexture(ItemStack stack, Entity entity, Equipme
      *
      * @param entity The entity swinging the item.
      * @return True to cancel any further processing by EntityLiving
+     * @deprecated To be replaced with hand sensitive version in 21.2
+     * @see #onEntitySwing(ItemStack, LivingEntity, InteractionHand)
      */
     default boolean onEntitySwing(ItemStack stack, LivingEntity entity) {
```

Comment:
Missing deprecation annotation, IDEs may not consider the javadoc for deprecation warnings

Reply:
Ahh good spot, thought I had annotated them all correctly